xml_converter/src/parseable.cpp

```diff
@@ -35,4 +35,10 @@ bool Parseable::init_xml_attribute(rapidxml::xml_attribute<>* attribute, vector<
     // I removed all of the offending variables. This whole section will be relooked at. 
     string item = normalize(get_attribute_name(attribute));
     return true;
-}
\ No newline at end of file
+}
+
+vector<string> Parseable::as_xml() const{
+    vector<string> result;
+    result.push_back("a");
```

Comment:
Lets just `panic` and freak out here instead of actually being a nicely behaved function. Look into something like `assert(false)` or throwing an error or `exit(123245)` or something.

Reply:
Exit failure added. Function still needs to return something to compile.